translations/expansion_content.tex/pl.po

```diff
@@ -153,6 +163,12 @@ msgid ""
 "War machines are also more expensive at the Trading Post.\n"
 "\n"
 msgstr ""
+"\\subsection*{\\hypertarget{War Machines}{Machiny Wojenne}}\n"
+"Dodano w rozszerzeniu Bastion.\n"
```

Comment:
Dodano czy dodane?

Reply:
No tu też miałem problem... bo:

1. słowo poprzedzone jest nagłówkiem Machiny Wojenne
2. Zdanie jest wprowadzeniem do tekstu "następnego" i w zasadzie w mojej opinii różnica polega tylko w tym do którego "elementu" przypiszemy pierwsze zdanie. Jeśli chcemy zachować logiczną ciągłość od nagłówka/śródtytułu to lepiej by było zostawić formę w odniesieniu do Rozszerzenia - czyli Dodano, natomiast jeśli w odniesieniu do kart i treści następującego tekstu - to wtedy Dodane

Reply:
w Wydarzeniach i Kartach Ulepszonych Statystyk zdecydowałeś się na wersję "dodane"

wg mnie to lepiej brzmi bo odnosi się do zawartości. Czyli Machiny Wojenne - one zostały dodane, tak samo Karty Ulepszonych Statystyk, Wydarzenia

Reply:
A widzisz za dużo tych Dodan(e/o) - czytam z pamięci... dobra by było mniej zmian robimy Dodane :)